boards/arm/nrf9160_pca10090/Kconfig.defconfig

```diff
@@ -30,4 +30,10 @@ config SPI_3
 
 endif # SPI
 
+config FLASH_LOAD_OFFSET
+	default $(dt_hex_val,DT_CODE_PARTITION_OFFSET) if TRUSTED_EXECUTION_NONSECURE
+
+config FLASH_LOAD_SIZE
```

Comment:
It would be nice to explain with a comment, why this Kconfigs are required for the Non-Secure build.

For example we need to clarify that we do not have any more DTS overlays (as we had for MCUboot) and so the linker gets the OFFSET and SIZE from Kconfig (which needs _this kind of_ setup, to obtain it from DTS)

Reply:
And same for _OFFSET of course.

Reply:
Updated

Reply:
Text is nice.